server/rpc/handle_get_tx_out.go

```diff
@@ -109,10 +99,9 @@ func handleGetTxOut(s *Server, cmd interface{}, closeChan <-chan struct{}) (inte
 	}
 
 	txOutReply := &rpcmodel.GetTxOutResult{
-		SelectedTip:   selectedTipHash,
-		Confirmations: confirmations,
```

Comment:
Since you are taking the data from the utxo-set, can't you calculate confirmations?
(Same things for other places where you deleted confirmations out of rpc results)/